x-pack/plugins/uptime/public/components/monitor/ml/ml_manage_job.test.tsx

```diff
@@ -6,35 +6,97 @@
+      expect(anomalyDetectionBtn).toBeInTheDocument();
+      userEvent.click(anomalyDetectionBtn as HTMLElement);
+
+      expect(forNearestButton(getByText)(labels.ENABLE_ANOMALY_ALERT)).toBeEnabled();
+    });
+
+    it('does not display an informative tooltip', async () => {
+      const { getByText, findByText } = render(
+        <ManageMLJobComponent hasMLJob={true} onEnableJob={jest.fn()} onJobDelete={jest.fn()} />,
+        {
+          state: makeMlCapabilities(),
+          core: makeUptimePermissionsCore({ save: true }),
+        }
+      );
+
+      const anomalyDetectionBtn = forNearestButton(getByText)(labels.ANOMALY_DETECTION);
+      expect(anomalyDetectionBtn).toBeInTheDocument();
+      userEvent.click(anomalyDetectionBtn as HTMLElement);
+
+      userEvent.hover(getByText(labels.ENABLE_ANOMALY_ALERT));
+
+      await expect(() =>
+        findByText('You need write access to Uptime to create anomaly alerts.')
+      ).rejects.toEqual(expect.anything());
+    });
   });
 
-  it('renders without errors', () => {
-    jest.spyOn(redux, 'useDispatch').mockReturnValue(jest.fn());
-    jest.spyOn(redux, 'useSelector').mockReturnValue(true);
-
-    const wrapper = renderWithRouter(
-      <KibanaContextProvider
-        services={{ ...core, triggersActionsUi: { getEditAlertFlyout: jest.fn() } }}
-      >
-        <ManageMLJobComponent hasMLJob={true} onEnableJob={jest.fn()} onJobDelete={jest.fn()} />
-      </KibanaContextProvider>
-    );
-    expect(wrapper).toMatchSnapshot();
+  describe("when users don't have write access to uptime", () => {
+    it('disables the button to create alerts', () => {
+      const { getByText } = render(
+        <ManageMLJobComponent hasMLJob={true} onEnableJob={jest.fn()} onJobDelete={jest.fn()} />,
+        {
+          state: makeMlCapabilities(),
+          core: makeUptimePermissionsCore({ save: false }),
+        }
+      );
+
+      const anomalyDetectionBtn = forNearestButton(getByText)(labels.ANOMALY_DETECTION);
+      expect(anomalyDetectionBtn).toBeInTheDocument();
+      userEvent.click(anomalyDetectionBtn as HTMLElement);
+
+      expect(forNearestButton(getByText)(labels.ENABLE_ANOMALY_ALERT)).toBeDisabled();
+    });
+
+    it('displays an informative tooltip', async () => {
+      const { getByText, findByText } = render(
+        <ManageMLJobComponent hasMLJob={true} onEnableJob={jest.fn()} onJobDelete={jest.fn()} />,
+        {
+          state: makeMlCapabilities(),
+          core: makeUptimePermissionsCore({ save: false }),
+        }
+      );
+
+      const anomalyDetectionBtn = forNearestButton(getByText)(labels.ANOMALY_DETECTION);
+      expect(anomalyDetectionBtn).toBeInTheDocument();
+      userEvent.click(anomalyDetectionBtn as HTMLElement);
+
+      userEvent.hover(getByText(labels.ENABLE_ANOMALY_ALERT));
+
+      expect(
+        await findByText('You need write access to Uptime to create anomaly alerts.')
```

Comment:
```suggestion
        await findByText('You need read-write access to Uptime to create anomaly rules.')
```

Reply:
Hmmm, actually having thought again about this:

I was thinking that perhaps the change from alerts to rules would actually have a different meaning from what the button does.

In that PR we're disabling the ability to create alerts themselves, as the rule is already created when users get to see that tooltip.

I will keep "alerts" as we discussed on Slack, thank you @EamonnTP!